client/android/div-core/src/main/java/com/yandex/div/core/images/DivImageDownloadCallback.java

```diff
@@ -16,6 +18,11 @@ public void onSuccess(@NonNull CachedBitmap cachedBitmap) {
         // no implementation
     }
 
+    @UiThread
+    public void onSuccess(@NonNull Drawable drawable) {
```

Comment:
KDoc comment will help to easily distinguish `onSuccess` methods. Can you add one describing the purpose and cases when this method will be called?